pkgs/by-name/cp/cpuinfo/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  stdenv,
+  gtest,
+  gbenchmark,
+  nix-update-script,
+  testers,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cpuinfo";
+  version = "0-unstable-2024-06-02";
+
+  src = fetchFromGitHub {
+    owner = "pytorch";
+    repo = "cpuinfo";
+    rev = "05332fd802d9109a2a151ec32154b107c1e5caf9";
+    hash = "sha256-VhTRHpT+4g97m+amOZ52lJWavHsN5QVnjnEn6wJzE3A=";
+  };
+
+  passthru.updateScript = nix-update-script {extraArgs = ["--version=branch"];};
+
+  passthru.tests.pkg-config = testers.testMetaPkgConfig finalAttrs.finalPackage;
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    gtest
+    gbenchmark
+  ];
+
+  cmakeFlags = [
+    "-DUSE_SYSTEM_GOOGLETEST=1"
+    "-DUSE_SYSTEM_GOOGLEBENCHMARK=1"
+  ];
```

```suggestion
    (lib.cmakeBool "CPUINFO_BUILD_BENCHMARKS" false)
  ];
```
I don't think we need to execute a benchmark when we package.
https://github.com/pytorch/cpuinfo/blob/05332fd802d9109a2a151ec32154b107c1e5caf9/CMakeLists.txt#L24